Library/Formula/globus-toolkit.rb

```diff
@@ -12,14 +10,22 @@ class GlobusToolkit < Formula
     sha1 "1cbbe7a3d08711cbc5665e91558417d7b2028d3a" => :mountain_lion
   end
 
+  depends_on "openssl"
   depends_on "libtool" => :run
+  depends_on "pkg-config" => :build
+
+  option "with-check", "Test the toolkit when installing"
 
   def install
     ENV.deparallelize
     ENV["MACOSX_DEPLOYMENT_TARGET"] = MacOS.version
-    system "./configure", "--prefix=#{prefix}"
+    man.mkpath
+    system "./configure", "--prefix=#{prefix}",
+                          "--mandir=#{man}",
+                          "--disable-dependency-tracking"
     system "make"
-    system "make install"
-    share.install prefix/'man'
+    system "make", "check" if build.with? "check"
+    system "make", "install"
+    share.install prefix/"man"
```

Comment:
@DomT4 That fixed it \o/
It now fails on the audit part:

```
Error: 2 problems in 1 formulae
globus-toolkit:
 * Non-executables were installed to "/usr/local/Cellar/globus-toolkit/6.0/bin"
The offending files are:
  /usr/local/Cellar/globus-toolkit/6.0/bin/ssh.d
 * Header files that shadow system header files were installed to "/usr/local/Cellar/globus-toolkit/6.0/include"
The offending files are:
  /usr/local/Cellar/globus-toolkit/6.0/include/gssapi.h
```

I’m currently debugging another formula update, will check this one after.